package.json

```diff
@@ -103,7 +103,7 @@
     "lokka-transport-http": "1.6.1",
     "mocha": "3.2.0",
     "moment": "2.29.4",
-    "mongodb": "4.10",
+    "mongodb-legacy": "6.1.3",
```

Comment:
Allows us to stick with callback based methods, for now. We will eventually need to refactor code to use all promises or `async/await`.  

https://www.npmjs.com/package/mongodb-legacy